Tropicalytics.podspec

```diff
@@ -29,7 +29,6 @@ Tropicalytics is a lightweight configurable analytics tool that allows you to se
     'Tropicalytics' => ['Pod/Assets/*.png']
   }
 
-  s.public_header_files = 'Pod/Classes/**/*.h'
```

Comment:
I believe this is used so other projects have access to the header files inside of Tropicalytics and not just the main file.